src/libs/actions/Report.js

```diff
@@ -940,6 +943,7 @@ function fetchAllReports(
         })
         .then((returnedReports) => {
             Onyx.set(ONYXKEYS.INITIAL_REPORT_DATA_LOADED, true);
+            isReportDataFresh = true;
```

Comment:
I think this flag is not needed at all. 

This key `ONYXKEYS.INITIAL_REPORT_DATA_LOADED` can be used to get the same info.

Reply:
I already tried but it is not the same. That key is stored in Onyx, so it is also cached on the device and only has a `false` value the first time a user logs in. If the user opens a new tab and is already logged in, that key is true even before requesting for updated reports. That is why I think it is necessary.

Reply:
Ah. agree.